data-management/data-interface.js

```diff
@@ -502,7 +505,8 @@ class DataInterface {
         const uuid = v4();
         const accessToken = createToken({...userInfo, uuid}, config.token_secret, config.token_timeout);
         const aUser = await this.dataService.linkTokenToUser({uuid, expiration: config.token_timeout}, userInfo);
-        if (aUser) await this.eventLoggingService.logCreateToken(new User(aUser.userID, aUser.email, aUser.IDP), new Token(uuid, config.token_timeout));
+        const token = new Token(uuid, addSeconds(getTimeNow(), config.token_timeout));

```

Comment:
Let's log the expiration here as an Epoch timestamp so that it consistent with the event timestamp format we are using. We can wrap the "addSeconds" call with "Date.parse(addSeconds(getTimeNow(), config.token_timeout))" or update addSeconds to return it in the correct format